src/test/java/io/opentracing/contrib/di/BraveTracerModule.java

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright 2017 The OpenTracing Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.opentracing.contrib.di;
+
+import brave.opentracing.BraveTracer;
+import com.google.inject.AbstractModule;
+import com.google.inject.Provides;
+import io.opentracing.Tracer;
+import zipkin.reporter.AsyncReporter;
+import zipkin.reporter.okhttp3.OkHttpSender;
+
+import javax.inject.Named;
+import javax.inject.Singleton;
+
+public class BraveTracerModule extends AbstractModule {
```

Comment:
I would like to keep samples in test (at least for the begining), to avoid split the project into a heavy multi-modules (project, especially with maven).
* I often look at the tests and samples as how to use a lib.
* having sample with test, keep it as compilable and easy to run sample (from IDE), like for test.